charts/zabbix/README.md

```diff
@@ -9,7 +9,7 @@ This Helm chart installs [Zabbix](https://www.zabbix.com) in a Kubernetes cluste
 
 # Prerequisites
 
-- Kubernetes cluster 1.10+
+- Kubernetes cluster 1.21+
```

Comment:
@fibbs you need change this in README.go.tmpl file

Reply:
indeed. Will change that